visivo/models/props/insight_props.py

```diff
@@ -0,0 +1,50 @@
+from enum import Enum
```

Comment:
I think these should be the same as TraceProps correct?  You are doing it for clarity and to be able to remove TraceProps?